### ISSUE TITLE
.NET Standard 2.0 support (with sample project)

### ISSUE THREAD

**Issue by Sergio0694** [ins0mniaque/ILSupport#17]
Hello, are there any plans for .NET Standard 2.0 support?
I mean both the support for it and the new .csproj format, as well as a sample project using .NET Standard 2.0.

Also, VS2017 currently gives an error when trying to create a PCL from the samples (as PCLs are no longer supported in VS2017 I think).

Thank you, keep up the good work! 😄

**Comment by 1996v** [comment]
:)

**Comment by jeffmccormick** [comment]
@Sergio0694 I forked the project and create compatibility for .NET Standard/Core over at https://github.com/jmccormi7/ILSupport

The updated `.targets` can be copied directly from here: https://raw.githubusercontent.com/jmccormi7/ILSupport/master/IL%20Support.ProjectTemplates/IL%20Support.Standard.targets

Currently waiting on a pull request back to this repository

**Comment by ins0mniaque** [comment]
This looks good. I'm swamped right now, but I'll try to pull your code and publish a new version later this week.

Thank you for your work!

**Comment by jeffmccormick** [comment]
@ins0mniaque the project templates still need some testing and tweaking, but I have verified that the new `.targets` works on a few projects.  I will try to test the project templates tomorrow.  Might be worth waiting to publish a new version until everything has been fully tested.

EDIT: Templates have been updated and tested, documentation updated.  Pull request is ready.

**Comment by nietras** [comment]
This sounds great :) Any progress with this?

**Comment by jeffmccormick** [comment]
@nietras yes, .NET Standard support was merged and is available with version 2.4 of the extension
